recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "cfanalytics" %}
-{% set version = "0.1.4" %}
+{% set version = "0.1.6" %}
 {% set sha256 = "aa3d98ebc1c39000c1e7ab08d586e079f8d7c87db15a318e03ff4918bc38c739" %}
```

Comment:
You need to update the checksum for the package.
You can get it here: https://pypi.org/project/cfanalytics/#files